### PR TITLE
Adding IsMeteredSupport functionality to database

### PR DESCRIPTION
**Pull request by msalemcode**
This PR will implement IsMeteredSupport fag automatic and eliminate the manual steps to enable the flag.

The PR also fix IsPerUser function to pass the new quantity for  subscription update

### CODE REVIEW

File: src/SaaS.SDK.Services/Services/FulfillmentApiService.cs [hunk 1]
```diff
@@ -197,7 +197,7 @@ public async Task<SubscriptionUpdateResult> ChangeQuantityForSubscriptionAsync(G
             {
                 try 
                 { 
-                    var operationId = await this.marketplaceClient.Fulfillment.UpdateSubscriptionAsync(subscriptionId, new SubscriberPlan { PlanId = "" });
+                    var operationId = await this.marketplaceClient.Fulfillment.UpdateSubscriptionAsync(subscriptionId, new SubscriberPlan { Quantity = subscriptionQuantity });
```
Comment (santhoshb-msft) [inline src/SaaS.SDK.Services/Services/FulfillmentApiService.cs:200]: might be a good idea to keep it in different commit?

Comment (msalemcode) [reply src/SaaS.SDK.Services/Services/FulfillmentApiService.cs]: Deferring the change to another PR.

File: src/SaaS.SDK.Services/Models/PlanDetailResult.cs [hunk 2]
```diff
@@ -48,5 +63,104 @@ public class PlanDetailResult : SaaSApiResult
...
+        public  PlanComponents PlanComponents { get; set; }
+
+
+        /// <summary>
+        /// Get  IsmeteringSupported associate with the plan.
+        /// </summary>
+        public bool? IsmeteringSupported { 
+            get
+            {
+                return m_IsmeteringSupported;
```
Comment (santhoshb-msft) [inline src/SaaS.SDK.Services/Models/PlanDetailResult.cs:133]: this.PlanComponents.MeteringDimensions.Count > 0

Comment (msalemcode) [reply src/SaaS.SDK.Services/Models/PlanDetailResult.cs:133]: the logic is covered under the method public List<MeteredDimensions> GetmeteredDimensions()

**Review by santhoshb-msft — APPROVED** [review]
All good from me!
